src/game/chopper.js

```diff
@@ -34,9 +37,22 @@ export default ex.Actor.extend({
     this.addDrawing('up', this.upAnimation);
     this.addDrawing('down', this.downAnimation);
 
+    this.on('collisionstart', (event) => {
+      if (event.other) {
+        const crashedObject = event.other;
```

Comment:
Perhaps move this logic into `crashedObject` and call it from here? Like `crashedObject.boom()`